imports/plugins/core/taxes/server/api/import.js

```diff
@@ -1,5 +1,5 @@
 import { Reaction } from "/server/api";
-import Import from "/server/api/core/import";
+import { Import } from "/server/api/core/import";
 import * as Collections from "../../lib/collections";
```

Comment:
Agree, that's bugged me for a while.  I created https://github.com/reactioncommerce/reaction/issues/1364 to track.